crates/primitives-traits/src/transaction/signed.rs

```diff
@@ -32,7 +33,7 @@ pub trait SignedTransaction:
     + alloy_rlp::Decodable
     + Encodable2718
     + Decodable2718
-    + TransactionExt
```

Comment:
since trait `TransactionExt` shrunk to basically only this method which isn't called on `TransactionSigned`, its redundant as super trait